.github/workflows/ci.yml

```diff
@@ -43,3 +43,11 @@ jobs:
             -Dsonar.projectKey=adobe_aem-spa-component-mapping
             -Dsonar.sources=src
             -Dsonar.tests=test
+      - name: Build documentation
+        run: npm install && npm run docs
+      - name: Publish documentation to github pages
+        uses: JamesIves/github-pages-deploy-action@3.5.9
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH: gh-pages-documentation
+          FOLDER: docs
```

Comment:
i would rather suggest performing those steps in https://github.com/adobe/aem-spa-component-mapping/blob/master/.github/workflows/release.yml - so on merge to `master`, rather than on `pull request`, wdyt?

Reply:
yes, these will be moved to release action, it's here for tests only - I don't want to create dummy releases for the tests